Tests/OctoKitTests/ReleasesTests.swift

```diff
@@ -11,12 +11,53 @@ import OctoKit
 
 final class ReleasesTests: XCTestCase {
     static var allTests = [
+        ("testListReleases", testListReleases),
         ("testPostRelease", testPostRelease),
         ("testLinuxTestSuiteIncludesAllTests", testLinuxTestSuiteIncludesAllTests)
     ]
 
     // MARK: Actual Request tests
 
+    func testListReleases() {
+        let session = OctoKitURLTestSession(expectedURL: "https://api.github.com/repos/octocat/Hello-World/releases", expectedHTTPMethod: "GET", jsonFile: "Fixtures/releases", statusCode: 200)
```

Comment:
*nitpick*: please split this to multiple lines to reduce the length of this line for easier review.